oxo.py

```diff
@@ -1,29 +1,159 @@
 class OxoBoard:
+
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        self.board_rows = 3
+        self.board_columns = 3
+        self.board_array = []
+        # create the board array
+        for x in range(self.board_rows):
+            self.board_array.append([])
+
+            for y in range(self.board_columns):
+                self.board_array[x].append(0)
+
+        self.show()
 
     def get_square(self, x, y):
         """ Return 0, 1 or 2 depending on the contents of the specified square. """
-        raise NotImplementedError("TODO: implement get_square")
+
+        square_content = self.board_array[x][y]
+        return square_content
```

Comment:
No real need for a temporary variable here -- could just `return self.board_array[x][y]`